File: backend-ms1/src/main/java/ma/zs/stocky/service/facade/admin/commande/CommandeAdminService.java
package ma.zs.stocky.service.facade.admin.commande;

import java.util.List;
import ma.zs.stocky.bean.core.commande.Commande;
import ma.zs.stocky.dao.criteria.core.commande.CommandeCriteria;
import ma.zs.stocky.zynerator.service.IService;



public interface CommandeAdminService extends  IService<Commande,CommandeCriteria>  {

    List<Commande> findByEntiteAdminId(Long id);
    int deleteByEntiteAdminId(Long id);
    long countByEntiteAdminCode(String code);




}

File: frontend/src/app/shared/service/admin/commande/CommandeAdmin.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from '@angular/common/http';
import {BehaviorSubject, Observable} from 'rxjs';

import {environment} from 'src/environments/environment';

import {CommandeDto} from 'src/app/shared/model/commande/Commande.model';
import {CommandeCriteria} from 'src/app/shared/criteria/commande/CommandeCriteria.model';
import {AbstractService} from 'src/app/zynerator/service/AbstractService';


@Injectable({
  providedIn: 'root'
})
export class CommandeAdminService extends AbstractService<CommandeDto, CommandeCriteria> {
     constructor(private http: HttpClient) {
        super();
        this.setHttp(http);
    }

    public constrcutDto(): CommandeDto {
        return new CommandeDto();
    }

    public constrcutCriteria(): CommandeCriteria {
        return new CommandeCriteria();
    }

    get API() {
        return environment.apiUrlStockyservice + 'admin/commande/';
    }
}
